packages/grafana-ui/src/components/NodeGraph/ViewControls.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Button } from '../Button';
+import { stylesFactory, useTheme } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { css } from 'emotion';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  minusButton: css`
+    margin-left: 2px;
+  `,
+
+  scale: css`
+    font-size: 10px;
+    color: ${theme.colors.textFaint};
+  `,
+
+  scrollHelp: css`
+    font-size: 8px;
+    color: ${theme.colors.textFaint};
+  `,
+}));
+
+interface Props<Config> {
+  config: Config;
+  onConfigChange: (config: Config) => void;
+  onPlus: () => void;
+  onMinus: () => void;
+  scale: number;
+}
+
+/**
+ * Control buttons for zoom but also some layout config inputs mainly for debugging.
+ */
+export function ViewControls<Config extends Record<string, any>>(props: Props<Config>) {
+  const { config, onConfigChange, onPlus, onMinus, scale } = props;
+  const [showConfig, setShowConfig] = useState(false);
+  const styles = getStyles(useTheme());
+
+  // For debugging the layout, should be removed here and maybe moved to panel config later on
+  const allowConfiguration = false;
+
+  return (
+    <>
+      <Button icon={'plus-circle'} onClick={onPlus} size={'sm'} title={'Zoom in'} />
```

Comment:
One thing that confused me a bit was that when reaching the maximum zoom in/out, clicking the buttons won't do anything. Definitely not blocking, but would be nice to disable them when reaching max/min scale value